pkg/application/service/service.go

```diff
@@ -3,11 +3,11 @@ package service
 import (
 	"context"
 	"fmt"
+	"github.com/horizoncd/horizon/pkg/param/managerparam"
 
-	applicationmanager "github.com/horizoncd/horizon/pkg/application/manager"
+	"github.com/horizoncd/horizon/pkg/application/manager"
 	"github.com/horizoncd/horizon/pkg/application/models"
-	groupservice "github.com/horizoncd/horizon/pkg/group/service"
-	"github.com/horizoncd/horizon/pkg/param/managerparam"
+	groupsvc "github.com/horizoncd/horizon/pkg/group/service"
 )
 
 type Service interface {
```

Comment:
Can rename `Servicer`

Reply:
there already service struct and interface in this pkg